common/lib/dependabot/dependency_group.rb

```diff
@@ -1,11 +1,20 @@
 # frozen_string_literal: true
 
+require 'wildcard_matcher'
+
 module Dependabot
   class DependencyGroup
-    attr_reader :name
+    attr_reader :name, :rules, :dependencies
 
-    def initialize(name)
+    def initialize(name, rule)
       @name = name
+      @rules = rule
+      @dependencies = []
+    end
```

Comment:
```suggestion
    def initialize(name, rules)
      @name = name
      @rules = rules
      @dependencies = []
    end
```

Reply:
I was just double-checking we were using positional arguments here, I'm going to update this to use keyword arguments in a PR I'd like to get through to the API layer so we don't have to change the contract later, but it's likely to conflict with this.

I'll link shortly.

Reply:
CRF: https://github.com/dependabot/dependabot-core/pull/7084

We are inconsistently using the group name as a keyword arg in some places, so let's consolidate on using keyword arguments as we start to upstream this code into the service.

Reply:
fixed in 99a5c63423f5c15282a2414554538958b829bb2c